### DISCUSSION TITLE
OAuth2 Bearer Token Usage | George Aristy

### DISCUSSION THREAD

**Discussion by giscus** [llorllale/llorllale.github.io#54] [Announcements]
# OAuth2 Bearer Token Usage | George Aristy

No matter what, as an implementer always verify your understanding of a specification against other implementations.

https://georgearisty.dev/posts/oauth2-token-bearer-usage/

**Comment by jimisola** [comment]
Thank you for the post. I just came over this issue with GitLab. Turns out both GitLab and JFrog expects non-base64 encoded OAuth Bearer tokens.
